packages/@sanity/base/src/components/collapseMenu/CollapseMenu.tsx

```diff
@@ -26,76 +28,198 @@ import {CollapseMenuDivider, CollapseMenuButton, CollapseMenuItemProps} from '.'
 
 interface CollapseMenuProps {
   children: React.ReactNode
+  collapsed?: boolean
   gap?: number | number[]
   menuButton?: ReactElement<HTMLButtonElement>
   menuPopoverProps?: PopoverProps
+  onMenuClose?: () => void
   onMenuVisible?: (visible: boolean) => void
+  disableRestoreFocusOnClose?: boolean
 }
 
-const Root = styled(Box)<{$hide?: boolean}>`
+const RootBox = styled(Box)<{$hide?: boolean}>`
   border-radius: inherit;
-  overflow: hidden;
   position: relative;
-  white-space: nowrap;
-  width: 100%;
-  padding: 0.25rem;
-  margin: -0.25rem;
+  overflow: hidden;
+  /**
+  * We need to add padding + negative margin to make the focusRing  visible
+  */
+  padding: ${({theme}) => theme.sanity.focusRing.width}px;
+  margin: -${({theme}) => theme.sanity.focusRing.width}px;
 `
 
-const Inner = styled(Flex)<{$hide?: boolean}>`
+const InnerFlex = styled(Flex)<{$hide?: boolean}>`
   position: absolute;
   top: 0;
   left: 0;
   right: 0;
   bottom: 0;
+  overflow: hidden;
+  white-space: nowrap;
   pointer-events: ${({$hide}) => ($hide ? 'none' : 'inherit')};
   opacity: ${({$hide}) => ($hide ? 0 : 1)};
   position: ${({$hide}) => ($hide ? 'absolute' : 'static')};
   visibility: ${({$hide}) => ($hide ? 'hidden' : 'visible')};
   width: ${({$hide}) => ($hide ? 'max-content' : 'auto')};
+  /**
+  * We need to add padding + negative margin to make the focusRing  visible
+  */
+  padding: ${({theme}) => theme.sanity.focusRing.width}px;
+  margin: -${({theme}) => theme.sanity.focusRing.width}px;
 `
 
 const OptionBox = styled(Box)<{$inView: boolean}>`
```

Comment:
It would be more correct to use `styled(Flex)` instead of `styled(Box)` here

Reply:
Hm, why? It seems there are no elements that needs a flex layout inside `<OptionBox>` below?